stout/flags.h

```diff
@@ -53,7 +53,7 @@ class Parser {
 
   // Helper for parsing a normalized form of flags.
   void Parse(
-      const std::multimap<std::string, std::optional<std::string>>& values);
+      std::multimap<std::string, std::optional<std::string>>& values);
```

Comment:
Instead of modifying `values` here let's modify it in the other version of `Parse()` and still call into this function with a `const &`.